README.md

```diff
@@ -1,67 +1,91 @@
-
-### Motivation
-
-Read [Why fix Kubernetes and Systemd](https://medium.com/@kris-nova/why-fix-kubernetes-and-systemd-782840e50104) by [Kris Nóva](https://github.com/krisnova).
-
-Aurae attempts to simplify and improve the stack in enterprise distributed systems by carving out a small portion of responsibility while offering a few basic guarantees with regard to state, synchronicity, awareness, and security.
-
-Aurae brings enterprise identity as low as the socket layer in a system, which unlocks multi tenant workloads that run below tools like Kubernetes.
-
-## AuraeScript
-
-Aurae offers a Turing complete scripting language built on top of TypeScript called [AuraeScript](https://github.com/aurae-runtime/aurae/tree/main/auraescript). AuraeScript embeds the [Deno](https://deno.land) source code directly, and offers a remote client and SDK to interface directly with Aurae remotely. The AuraeScript library is automatically generated from the `.proto` files defined in the [Aurae Standard Library](https://aurae.io/stdlib/).
-
-Valid TypeScript files can be leveraged to replace static manifests, such as YAML, as well as interact directly with a running system.
+<!-- THE DOCUMENT -->
+
+![Deploy (aurae.io)] ![Documentation] ![Main Build (lint, compile, test)]
+
+# Summary
+
+[Aurae] is a free and [open-source] project built with the [Rust] systems
+language.
+
+The project adapts to the expanding demands of enterprise distributed systems
+that meets the present and future demands for brilliant higher-order distributed
+systems.
+
+Aurae supports multi-tenant workloads and enterprise identities all the way down
+to the socket layer and can rationalize workloads and further improve system
+proportions by utilizing a variety of isolation strategies for enterprise
+workloads, the project can work exceptionally well with tools like Kubernetes [^compare].
+
+## Project Status
+
+> **STILL IN EARLY DEVELOPMENT!**<br>
+> **The Aurae project and API can change without notice.**<br> 
+> **Do not run the project in production until further notice!**
+> <br><br>
+> _"Aurae attempts to simplify and improve the stack in_
+> _enterprise distributed systems by carving out a small portion of_
+> _responsibility while offering a few basic guarantees with regard to state,_
+> _synchronicity, awareness, and security."_
+> <br>&mdash; [Kris Nóva] (Founder of the Nivenly Foundation)
+
+_Auraes mission is to become the most loved method of managing workloads on a_
+_single piece of hardware._
+
+- The Aurae project welcomes contributions of all kinds and sizes.
+- Please read the "[getting involved]" documentation before contributing to the
+  project.
+- You do not have to know [Rust] to join the project.
+
+By joining the project in its early phases, you'll participate in the
+development of a milestone candidate for enterprise distributed systems and
+automation.
+
+# **Aurae**
```

Comment:
I don't understand this section? Is this just "more detail" or "expanded overview"? If so we should call it that. There isn't actually a binary or tool called "aurae" so maybe we shouldn't use it as a nebulous term/title?

Reply:
This is helpful. Changed it accordingly.